### PR TITLE
Add command for generating secret key

### PR DESCRIPTION
**Pull request by ttrig**

### CODE REVIEW

File: src/Commands/GenerateKey.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    use ConfirmableTrait;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'butler-auth:generate-key
+                    {--show : Display the key instead of modifying files}
+                    {--force : Force the operation to run when in production}';
```
Comment (emil-nasso) [inline src/Commands/GenerateKey.php]: This flag doesn't seem to be used?

Comment (ttrig) [reply src/Commands/GenerateKey.php]: It is, in `ConfirmableTrait`.

**Comment by olyckne** [comment]
Maybe add some tests for the command?

**Review by wecc — APPROVED** [review]

**Review by hejhansson — APPROVED** [review]

**Review by olyckne — APPROVED** [review]

**Review by ckarlbe — APPROVED** [review]

**Comment by ttrig** [comment]
Changed signature from `butler-auth:generate-key` to `butler-auth:generate-secret-key`.

**Review by hejhansson — APPROVED** [review]

**Review by wecc — APPROVED** [review]

**Review by emil-nasso — APPROVED** [review]

**Review by olyckne — APPROVED** [review]

**Pull request merged**